andela/travel_tool_back
# #160165626 Add options to decode the token for staging and production environments

#### What does this PR do?
Adds an option to decode the token for staging and production environments

#### Description of Task to be completed?
- add conditions for how the token is decoded on staging and production environments

#### How should this be manually tested?
- Navigate to https://travela-staging.andela.com/
- Login if you aren't logged in already
- You should see your requests if you have any
- You should be able to create a new request

#### Any background context you want to provide?
This PR needs to be merged before the fix reflects on staging

#### What are the relevant pivotal tracker stories?
[#160165626](https://www.pivotaltracker.com/story/show/160165626)

#### Screenshots (if appropriate)
None

#### Questions:
None
